### PR TITLE
Update button storybook examples

### PR DESCRIPTION
**Pull request by macandcheese**
Adds some examples to the button storybook..

Side note - anyone have a configuration in vs code that can format these files? No combination of language / formatter seems to do anything with storybook files :(

### CODE REVIEW

**Comment by paulcpederson** [comment]
I think you would need something that indents html inside template literals like https://marketplace.visualstudio.com/items?itemName=Tobermory.es6-string-html

File: src/components/calcite-button/calcite-button.stories.js [hunk 1]
```diff
@@ -46,22 +47,46 @@ storiesOf("Button", module)
     "With icon",
     () => `
     <calcite-button
-      icon="${text("icon", images24)}"
+      appearance="${select(
+        "appearance",
+        {
```
Comment (paulcpederson) [inline src/components/calcite-button/calcite-button.stories.js]: when the property is the same as the value the options can be an array like `["solid", "clear", "inline", "outline", "transparent"]` IIRC

**Comment by paulcpederson** [comment]
feel free to merge or play around with that shorter syntax for `select`

File: src/components/calcite-button/calcite-button.stories.js (hunk 1 above)
Comment (macandcheese) [reply src/components/calcite-button/calcite-button.stories.js]: Oh, rad. Will update ⚡️

**Pull request merged**